content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Manual Rotation of CA Certificates
+min-kubernetes-server-version: v1.13
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page shows how to manually rotate the certificate authority (CA) certificates.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
```

Comment:
Later on the page says: “Make sure container networking functions.”

I wonder, is it worth calling out the aptitudes that we assume the reader has (eg: able to validate container networking is working correctly)? There are certainly a few skills they will need.

Reply:
hmm I agree, I mentioned steps like that based on my prior experience. coredns pods running doesn't always means it is functional (few earlier bugs and scenarios). I can remove it too, but it would be better to mention what people must not forget to check.

For example, kubernetes documentation has reference to use dnsutils to validate dns services. That would actually validate container networking function in a way. People could use document to debug services https://kubernetes.io/docs/tasks/debug-application-cluster/debug-service/ as well.

So yes, they need few other skills, but this kind of activity has to be performed by people who know what exactly they would be doing and that includes a basic networking and some understanding of PKI to list just bare minimum.